src/database/scrape-attempt.js

```diff
@@ -1,8 +1,15 @@
 const { Schema } = require('mongoose');
 
 module.exports = new Schema({
```

Comment:
this model we would need to update to include list of pdfs field with { link, hash }

Reply:
`urls: [new Schema({ url: String, hash: String })],`

Reply:
hmm..i think we still want to separate out main link from pdf links. url: string, pdfs: [new Schema({url, hash})]